Test failure in CI build 9575

The following test appears to have failed:

[#9575](https://circleci.com/gh/cockroachdb/cockroach/9575):

```
I1124 15:14:51.219031 1014 gossip/gossip.go:172  gossiping node descriptor node_id:3 address:<network_field:"localhost" address_field:"3" > attrs:<> 
I1124 15:14:51.219311 1014 storage/queue.go:193  no system config available. skipping...
I1124 15:14:52.296656 1014 storage/engine/rocksdb.go:132  closing in-memory rocksdb instance
I1124 15:14:52.296788 1014 storage/engine/rocksdb.go:132  closing in-memory rocksdb instance
I1124 15:14:52.296848 1014 storage/engine/rocksdb.go:132  closing in-memory rocksdb instance
--- FAIL: TestStoreRangeUpReplicate (1.08s)
	client_raft_test.go:569: storage/client_raft_test.go:560: condition failed to evaluate true within 1s
=== RUN   TestStoreRangeDownReplicate
I1124 15:14:52.298075 1014 multiraft/multiraft.go:557  node 1 starting
I1124 15:14:52.298165 1014 storage/replica.go:1308  gossiping cluster id  from store 1, range 1
I1124 15:14:52.298329 1014 raft/raft.go:409  [group 1] 1 became follower at term 5
I1124 15:14:52.298374 1014 raft/raft.go:218  [group 1] newRaft 1 [peers: [1], term: 5, commit: 10, applied: 10, lastindex: 10, lastterm: 5]
I1124 15:14:52.298453 1014 multiraft/multiraft.go:959  node 1 campaigning because initial confstate is [1]
I1124 15:14:52.298568 1014 raft/raft.go:489  [group 1] 1 is starting a new election at term 5
I1124 15:14:52.298585 1014 raft/raft.go:422  [group 1] 1 became candidate at term 6
I1124 15:14:52.298598 1014 raft/raft.go:471  [group 1] 1 received vote from 1 at term 6
--
I1124 15:15:02.666020 1014 storage/replica.go:1369  gossiping system config from store 1, range 2
I1124 15:15:02.669164 1014 storage/engine/rocksdb.go:132  closing in-memory rocksdb instance
--- PASS: TestStoreSplitReadRace (0.02s)
=== RUN   Example_rebalancing
--- PASS: Example_rebalancing (0.11s)
FAIL
FAIL	github.com/cockroachdb/cockroach/storage	19.818s
=== RUN   TestBatchBasics
I1124 15:14:39.185882 991 storage/engine/rocksdb.go:132  closing in-memory rocksdb instance
--- PASS: TestBatchBasics (0.00s)
=== RUN   TestBatchGet
I1124 15:14:39.186950 991 storage/engine/rocksdb.go:132  closing in-memory rocksdb instance
--- PASS: TestBatchGet (0.00s)
=== RUN   TestBatchMerge
I1124 15:14:39.189451 991 storage/engine/rocksdb.go:132  closing in-memory rocksdb instance
--- PASS: TestBatchMerge (0.00s)
=== RUN   TestBatchProto

```
Please assign, take a look and update the issue accordingly.

--- a/storage/client_raft_test.go
+++ b/storage/client_raft_test.go
@@ -565,7 +565,7 @@ func TestStoreRangeUpReplicate(t *testing.T) {
 			}
 		}
 		return true
-	}, 1*time.Second); err != nil {
+	}, 3*time.Second); err != nil {
 		t.Fatal(err)
 	}
 }

